File: remove-bg.js
const { Jimp } = require("jimp");

Jimp.read("Logo-soorder.png").then(img => {
  img.scan(0, 0, img.bitmap.width, img.bitmap.height, function(x, y, idx) {
    const r = this.bitmap.data[idx];
    const g = this.bitmap.data[idx + 1];
    const b = this.bitmap.data[idx + 2];
    if (r < 30 && g < 30 && b < 30) {
      this.bitmap.data[idx + 3] = 0;
    }
  });
  return img.write("logo-transparent.png");
}).then(() => {
  console.log("Done — logo-transparent.png saved");
}).catch(err => {
  console.error(err);
  process.exit(1);
});

File: remove-bg.py
from PIL import Image

img = Image.open("Logo-soorder.png").convert("RGBA")
data = img.getdata()
new_data = []
for item in data:
    if item[0] < 30 and item[1] < 30 and item[2] < 30:
        new_data.append((0, 0, 0, 0))
    else:
        new_data.append(item)
img.putdata(new_data)
img.save("logo-transparent.png")
print("Done — logo-transparent.png saved")
